app/models/reports/export_raw.rb

```diff
@@ -86,6 +86,8 @@ def default_report_hash
         journal_date: ->(od) { od.journal.journal_date if od.journal },
         reconciled_note: :reconciled_note,
         reconciled_at: :reconciled_at,
+        price_change_reason: :price_change_reason,
+        price_changed_by_user: ->(od) { od.price_changed_by_user.try(:full_name) },
```

Comment:
I don't have a strong feeling on this, but what if we used `username`? We could also just rely on the `to_s` of `User`, though that includes `(Deactivated)` if the user is deactivated (only UIC has deactivated users currently, but NU does have that feature in the backlog).

Reply:
Is there a reason we would prefer username over full name? We use full name elsewhere for whodunnits, i.e. in Admin Reservations etc.